prow/prowjobs/GoogleCloudPlatform/k8s-cloud-provider/k8s-cloud-provider-config.yaml

```diff
@@ -0,0 +1,20 @@
+presubmits:
+  GoogleCloudPlatform/k8s-cloud-provider:
+  - name: pull-k8s-cloud-provider-test
+    branches:
+    - master
+    always_run: true         # Run for every PR, or only when requested.
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191213-55437e3-master
+        args:
+        - "--repo=github.com/GoogleCloudPlatform/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - --scenario=execute
+        - --
+        - make
+        - --
+        - test
```

Comment:
can switch to pod-utils with roughly
```yaml
presubmits:
  GoogleCloudPlatform/k8s-cloud-provider:
  - name: pull-k8s-cloud-provider-test
    branches:
    - master
    always_run: true         # Run for every PR, or only when requested.
    labels:
      preset-service-account: "true"
   decorate: true
    spec:
      containers:
      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191213-55437e3-master
        command:
        - make
        - --
        - test
```

Reply:
might want a `path_alias` to set the checkout to .../go/src/<alias>

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md
https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md

Reply:
sample https://github.com/kubernetes/test-infra/blob/b3462c342653e56c2b144ba1c7e7c98f169cb3c6/config/jobs/kubernetes/kops/kops-config.yaml#L15